templates/content/coc.html.jinja2

```diff
@@ -71,20 +71,20 @@
 </p>
 <h3>Bridging</h3>
 <p>
-    HackSoc’s primary chat platform is Slack. While it is not perfect, we have not yet found a suitable alternative. A summary of the alternatives as well as their pros and cons can be found <a href="https://wiki.hacksoc.org/wiki/Chat/Alternatives" target="_blank">here</a>. We additionally have two other main chat platforms, that are bridged to various channels on Slack.
+    HackSoc’s primary chat platform is Discord. It’s not perfect, but no platform is. A summary of the alternatives as well as their pros and cons can be found <a href="https://wiki.hacksoc.org/wiki/Chat/Alternatives" target="_blank">on our wiki</a>. We have two other chat platforms, Slack and IRC, that are bridged to various channels on Discord.
 </p>
 <p>
-    The HackSoc Discord server is primarily used for virtual events, although it is also frequently used by members as a way to hang out virtually outside of events. The bridged channels are: #general, #food, #bot-testing, and #gaming.
+    The HackSoc Slack workspace is primarily used by past members of HackSoc, but is still available to anyone who prefers its interface. The bridged channels are: #general, #random, #art, #cursed, #electronics, #f1, #food, #gaming, #music, #politics, #quotes, #wholesome, #bot-testing, and #discord-migration.
```

Comment:
it should probably be specified that #bot-testing is bridged to discord#beep-boop

Reply:
done